mango-core/src/main/java/org/calrissian/mango/types/TypeRegistry.java

```diff
@@ -117,7 +117,7 @@ public Object decode(String alias, U value) {
         if (encoder != null)
             return encoder.decode(value);
 
-        throw new TypeDecodingException("An unknown alias [" + value + "] was encountered");
+        throw new TypeDecodingException("An unknown alias [" + alias + "] was encountered while decoding value [" + value + "]. Valid aliases are " + String.join(", ", aliasMapping.keySet()));
```

Comment:
Two problems with this.

1. We shouldn't assume that the value is easily representable as a string.  There are instances for example where byte[] are used instead of strings.  These will print out as a cryptic error message.  That being said the error message was incorrectly using the value instead of the alias.